Makefile

```diff
@@ -28,8 +28,8 @@ FIRST_PGPORT ?= 5500
 
 TMUX_EXTRA_COMMANDS ?= ""
 TMUX_LAYOUT ?= even-vertical	# could be "tiled"
-TMUX_TOP_DIR = ./tmux
-TMUX_SCRIPT = ./tmux/script-$(FIRST_PGPORT).tmux
+TMUX_TOP_DIR = $(PWD)/tmux
```

Comment:
I think instead of $(PWD) we should use the classic GNU Make trick for the top-level directory:

``` make
TOP := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
```

Put that in the first line of the Makefile.

Reply:
> > The XDG paths were relative, so once I cded there the XDG paths
> > were suddenly wrong.
>
> I'm missing the fix for that, can't see that change in the PR?

This makes the XDG paths absolute for me. I now see that it's better to do it in the C code, so it also works if you provide your own TMUX_TOP_DIR. So I think I will undo this change and do it in the better way.